src/csvcubed/models/cube/qb/validationerrors.py

```diff
@@ -318,7 +320,119 @@ class EmptyQbMultiMeasureDimensionError(SpecificValidationError):
 
     @classmethod
     def get_error_url(cls) -> str:
-        return 'http://purl.org/csv-cubed/err/empty-multi-meas-dimension'
+        return "http://purl.org/csv-cubed/err/empty-multi-meas-dimension"
 
     def __post_init__(self):
-        self.message = 'The field attribute of a QbMultiMeasureDimension must be populated'
+        self.message = (
+            "The field attribute of a QbMultiMeasureDimension must be populated"
+        )
+
+
+@dataclass
+class PivotedShapeMeasureColumnsExistError(BothMeasureTypesDefinedError):
+    """
+    An error to inform the user that they have attempted to define a pivoted shape cube with measure columns.
+    """
+
+    @classmethod
+    def get_error_url(cls) -> str:
+        return "https://gss-cogs.github.io/csvcubed-docs/external/guides/errors/build-command-errors/multiple-measure-columns/"
+
+    def __post_init__(self):
+        self.message = f"The cube is in pivoted shape, but you have defined 1 or more Measure columns. These two approaches are incompatible."
+
+
+@dataclass
+class DuplicateMeasureError(SpecificValidationError):
+    """
+    An error to inform the user that the obesrvation colums contain duplicate values
```

Comment:
This description isn't quite right. Are you able to correct it?

Reply:
Corrected description